ostelco-ios-client/ViewControllers/Home/HomeViewController2.swift

```diff
@@ -46,13 +46,49 @@ class HomeViewController2: UIViewController {
         }
         didPullToRefresh()
     }
-    
+
+    // Make the string with all the styles required for the balance text
+    class func getStylizeBalanceString(text: String) -> NSMutableAttributedString? {
+        let textArray: [Substring] = text.split(whereSeparator: { $0 == " " })
+        let decimalSeparator = Locale.current.decimalSeparator!.first!
```

Comment:
why do we need the first character of the decimal separator instead of the whole separator?